app/views/cards/_form.html.erb

```diff
@@ -1,4 +1,11 @@
-<%= simple_form_for [current_project, @board, @list, @card] do |f| %>
+<%= simple_form_for [current_project, @board, @list, @card],
+  html: {
+    data: {
+      behavior: ('local-auto-save' if @card.new_record?),
+      auto_save_key: ("project-#{current_project.id}-board-#{@board.id}-list-#{@list.id}-card" if @card.new_record?)
```

Comment:
Is there an advantage for using a descriptive key? Ideally, wouldn't the `list.id` suffice here since we know it's unique across the projects?

Reply:
Thanks for pointing that out @aapomm , fixed!